hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/SstFilteringService.java

```diff
@@ -89,13 +89,17 @@ public class SstFilteringService extends BackgroundService
 
   private AtomicBoolean running;
 
-  private BooleanTriFunction<String, String, String, Boolean> filterFunction =
-      (first, last, prefix) -> {
-        String firstBucketKey = RocksDiffUtils.constructBucketKey(first);
-        String lastBucketKey = RocksDiffUtils.constructBucketKey(last);
-        return RocksDiffUtils
-            .isKeyWithPrefixPresent(prefix, firstBucketKey, lastBucketKey);
-      };
+  // Note: This filter only works till snapshots are readable only.
+  // In the future, if snapshots are changed to writable as well,
+  // this will need to be revisited.
+  static final BooleanTriFunction<String, String, String, Boolean>
```

Comment:
any reason to make it static? You can let it be a package private member. SSTFiltering service will have only single instance anyway.

Reply:
I did it for testing purpose. Otherwise I've to do lots of setup for testing. I think it is OK to keep it as static function because it is stateless. I can revert it and add get if you have a strong point to not use as static function.